packages/interchain/interchain-core/src/packet.rs

```diff
@@ -170,20 +170,25 @@ impl<Chain: CwEnv> IndexResponse for NestedPacketsFlow<Chain> {
 }
 
 pub mod success {
+    use crate::ack_parser::IbcHooksAck;
     use crate::{ack_parser::polytone_callback::Callback, tx::TxId};
     use cosmwasm_std::{Binary, Empty, StdError};
     use cw_orch_core::environment::CwEnv;
     use cw_orch_core::environment::IndexResponse;
 
     /// Contains the result (ack success) associated with various Ibc applications
     #[derive(Debug, PartialEq, Clone)]
+    #[non_exhaustive]
```

Comment:
Adding extra field and `non_exhaustive` should be breaking I suppose

Reply:
Yes exactly, but as mentionned in the PR description, I'm not sure we should care here, thoughts ?

Reply:
> Yes exactly, but as mentionned in the PR description, I'm not sure we should care here, thoughts ?

Yeah, sounds good, as long as this enum not used by any other cw-orch package